WalletWasabi.Gui/Converters/NetworkColorConverter.cs

```diff
@@ -0,0 +1,26 @@
+using Avalonia;
+using Avalonia.Data.Converters;
+using Avalonia.Media;
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using WalletWasabi.Exceptions;
+
+namespace WalletWasabi.Gui.Converters
+{
+	public class NetworkColorConverter : IValueConverter
+	{
+		public object Convert(object value, Type targetType, object parameter, CultureInfo culture) =>
+			Brush.Parse(value switch
+			{
+				Network network when network == Network.TestNet => "#318522",
+				Network network when network == Network.RegTest => "#AE6200",
+				_ => "#007ACC"
```

Comment:
@jmacato can we use the out theme file to store the colors? Like ApplicationAccentBrushLow?

Reply:
Yes i believe i commented that we could use resources to fetch

Reply:
Can you add that as code suggestions or example of how to do that?

Reply:
Is there something to get the color from the Styles through the resource manager?